services/server/youtube.ts

```diff
@@ -1,12 +1,12 @@
-type PlayListType = {
+export type PlayListType = {
```

Comment:
```suggestion
export type PlaylistType = {
```
playlist is one word (this needs to be fixed in a few places)

Reply:
fixed